### ISSUE TITLE
Staging Review Benefits 2: Combined Debt Portal

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#43302]
# NOTE
This template is being deprecated. If your kickoff meeting took place after May 26, 2022, you should be following the guidance specified in the Collaboration Cycle Rquest per the [May 26, 2022 change history post](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Change-history.1846870032.html) instead. If you are unsure if this applies to you, please reach out to the Governance Team on the vfs-platform-support Slack channel.

# Steps to complete Staging Review

## Before meeting

**VFS Product Manager**
- [x]   Create this issue and schedule meeting via Calendly (you are responsible for adding the VFS attendees).
- [x]   Notify the Collaboration Cycle team of this ticket and scheduled date/time in #vfs-platform-support ([see Slack Workflow (User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)).
- [x]  Add artifacts listed below to the ticket at least 4 days before scheduled meeting.
- [x]  Ensure you've provided staging access information for the tool or feature. (Don't put staging credentials in your va.gov-team ticket; store or reference them in a .md file in the va.gov-team-sensitive repository.)
- [ ]  If this product contains any [experimental design](https://design.va.gov/experimental-design/), add the experimental-design label and schedule a meeting with DSC to present the research findings.

 **Platform**
 
- [x] Add the Platform reviewers to scheduled meeting
- [x] Add Zoom information to scheduled meeting
- [ ] Create feedback tickets and link them to the Staging Review epic 

## After meeting

**VFS** 
- [ ] Respond to Platform feedback tickets.
- [ ] Close individual feedback tickets when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
- [ ] Update Staging Review ticket with the product’s launch date after it has gone live.
- [ ] Close Staging Review ticket when the review is complete and the product has launched.
- [ ] Communicate any and all known Accessibility issues that are not addressed prior to launch to the VA 508 Office via [Audit Request](https://depo-platform-documentation.scrollhelp.site/developer-docs/Instructions-for-VA-508-Office-document-review.1857913061.html#HowtorequestsupportfromtheVA508Office-AuditRequest). 

**Platform**

- [ ] [Update Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).

## Artifacts

Provide links or documents for the following: 

- [ ] Staging URL and staging test-user information
- [Staging URL - Combined Debt Portal](https://www.staging.va.gov/manage-debt-and-bills/summary)
- Staging Users:
  - StagingUser#81 has both a copay bill and benefit debt (*best to see full experience*)
  - StagingUser#228 has only benefit debts
  - StagingUser#80 has only medical copay bills
  - StagingUser#56 -has neither benefit debt nor copay bills
- [ALL Use Cases outlined here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/use_cases.md) 

- [ ] [Product outline - Combined Debt Portal](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/product_outline.md)
 
  

- [ ] [Release plan - Combined Debt Portal](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/release_plan.md)

 

### Content artifacts 

- [ ] Your product’s content source of truth
- [\[SW-Content\] Phase 1: Debt portal content and IA strategy  #36844](https://github.com/department-of-veterans-affairs/va.gov-team/issues/36844)
-  [\[Content & IA Design\] Combined debt portal  #39294](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39294)
- [IA Design for Combined Debt Portal](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/debt-portal.md)
- [ ] Github ticket for any relevant static content page and entry point updates for tool or feature (if applicable)

### QA artifacts

[Learn more about QA test plans](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)

- [x] Product test cases/test plan in TestRail with test execution logs
- [QA Test Plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42301) 
- [TestRail Test Plan](https://dsvavsp.testrail.io/index.php?/plans/view/3284)
- [x] [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/340) 
- [ ] [Summary(Defects) report in TestRail](https://dsvavsp.testrail.io/index.php?/reports/view/14) 
- [#42709](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42709)
- [#42710](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42710)
- [#42711](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42711)
- [#42785](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42785)

### Accessibility artifacts 

- [ ] [Completed TestRail Platform accessibility test plan](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc). **(Preferred)** OR, 

- [ ] List of known issues

     •	Issue 1
     •	Issue 2
     •	Issue 3

## Scheduling

- Open the [Calendly staging review calendar](https://calendly.com/collaboration-cycle/staging-review)
- Select a date and time and click “Confirm”
- Add your name and email
- Click "Add Guests" and enter the VFS meeting attendees email addresses. 
- Click Schedule Event

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Staging Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html) on Platform website for more information about the Collaboration Cycle.

 FYI: @shiragoodman , @chaseakirby

**Comment by allison0034** [comment]
@denisecoveyduc when I log in with 80 and 81 I get an error for copayments...

We can’t access your copay records right now

cc @shiragoodman @chaseakirby

**Comment by tlei123** [comment]
@ddzz , QA test-runs are currently incomplete.  Let me know when you'd need the Summary (Defects) report generated for you &mdash; it may be incomplete for this Friday's Staging Review mtg.

For current test-results, please just go to the [TR Test Plan](https://dsvavsp.testrail.io/index.php?/plans/view/3284).

Test-runs completed for 3 of the 5 test-cases so far.  The 2 outstanding cases' runs are currently pending:
- Correct Staging test-users TBD by App-Team; and
- [medical_copays endpoint error](https://github.com/department-of-veterans-affairs/va.gov-team/issues/42709#issuecomment-1167745797) fix [by Lighthouse? &mdash; @Dr-Pongo @OptionSelect , pls. confirm/update who needs to fix this].

I'll keep you posted over the next couple days.

cc @denisecoveyduc

**Comment by denisecoveyduc** [comment]
Hi everyone, Apologies I thought I had send a reply yesterday but looks like it didn't post :( 

@allison0034 Thank you for reaching out there is a known issue with  the copays on the staging environment. The problem does not exist on production. We thought it was going to be resolved earlier in the week but obviously the problem persists. 

@ScottyRJames has been working with the VBS team to resolve. We will follow up this morning to see if we can get a better handle on when that will be working again. @shiragoodman We will let you know what we uncover and then we can determine if we need to postpone the staging review. Sorry for the confusion.

Please note we have not changed the functionality within the copay tool in any way. The Combined Portal Summary page will display the # of copays and the total amount along with a link to the copay tool but the tool itself was not changed..  I am not sure if that changes the review process or not but wanted to make you aware of that. 

From a QA Testing perspective, as @tlei123 pointed out the QA test-run for Use Case C is held up by this VBS copay error and @OptionSelect  are looking for an alternative staging user for Use Case D. Hopefully we can provide Tze with what is needed later today. (_fingers crossed_)

**_Thanks everyone!_** As soon as we have an update I will comment again. 
cc: @Dr-Pongo @mgleo @chaseakirby

**Comment by denisecoveyduc** [comment]
Hi @allison0034 That is correct, Veterans are able to see copays on production without any issues. The problems only occur on Staging. We continue to monitor Sentry just in case. 

Here is the **[UXPin link](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/141834694/simulate/sitemap?mode=i)** that was shared on the **[staging review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/32081)** for the **[medical copay tool](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/Product_Outline_Medical_Copays.md)** which launched on January 13, 2022.  Hope that helps.

**Comment by ddzz** [comment]
@denisecoveyduc Do copay records work in staging? I keep seeing an error in the `Debt and bill overview` section.

**Comment by denisecoveyduc** [comment]
@ddzz  I have been told it will be fixed today. As soon as I hear officially can let you know. 
cc: @shiragoodman

**Comment by denisecoveyduc** [comment]
@shiragoodman It looks like that VBS error is still impacting our medical copay use cases and therefore @tlei123 is unable to complete the QA work. ☹️ I am guessing that means we need to bump our staging review out?

**Comment by chaseakirby** [comment]
@denisecoveyduc Hi Denise, jumping in since Shira is out today.
I agree that we should bump out the staging review so that QA and Accessibility can perform their QA and Accessibility tests with the functioning copays. I can work with you to reschedule for another time slot.
Thank you for keeping us posted on the status of the fix!

**Comment by denisecoveyduc** [comment]
Thanks @chaseakirby, Totally understand and thank you everyone for your patience.

**Comment by briandeconinck** [comment]
@denisecoveyduc When I follow the [staging URL](https://www.staging.va.gov/manage-debt-and-bills/summary) provided in this ticket, I get a "Sorry -- we can't find that page." Did the staging URL change?

**Comment by Dr-Pongo** [comment]
@briandeconinck , you need to be authenticated to hit that URL. We will eventually be going live on `/manage-va-debt` and taking the place of the existing debt portal. 

CC: @denisecoveyduc

**Comment by briandeconinck** [comment]
Ah, thank you @Dr-Pongo! Authenticate, then visit the staging URL, everything working as expected.

**Comment by tlei123** [comment]
@pjhill , yeah, e.g., I started generating a report at around 2:30 pm MT today and it's still in progress right now at almost 4 pm MT.

**Comment by tlei123** [comment]
^ 5:53 pm MT: Report generation still "in-progress."  I'll check again tmrw (Thu) morning.

**Comment by Dr-Pongo** [comment]
Hey @CherylEvans! It looks like that user is having some issues that may be outside of our control. User 1 is a valid alternative that should meet the requirements for Use Case 1. There is a known issue viewing the downloadable debt letter from VMBs with that user, but it is showing a good error for it and the test case doesn't seem to specify going that deep. 

CC @denisecoveyduc , @allison0034 
Thank you @ScottyRJames  @OptionSelect

**Comment by tlei123** [comment]
@pjhill @ddzz , sorry, **Summary (Defects) report** started yesterday ~ 2:30 pm MT still appears to be in progress right now at 8:55 am MT.

**Comment by denisecoveyduc** [comment]
The debt resolution team has resolved all staging review feedback for the Combined Debt Portal

**Comment by shiragoodman** [comment]
hey @denisecoveyduc  - I was doing some manual data collection today and noticed my issue count was off by 3. Turns out, the issues that Chase had identified above on July 7 (#44003, #44004, and #44005) were not the only additional issues we found. #44000, #44001 and #44002 were also created and somehow missed being added as child tickets to this epic (I'm thinking they were issues we discussed at the review meeting, which is why Chase didn't feel the need to specify that they were new).

I have added them as child tickets to this epic, and they are still open issues.

Please take a look and let me know if you have any questions. This epic can remain closed, but I just wanted to bring to your attention.

**Comment by denisecoveyduc** [comment]
@shiragoodman Thanks for highlighting these for us. We will have a look and ensure they have been addressed.
